Framework/Kernel/src/Material.cpp

```diff
@@ -72,8 +115,22 @@ double Material::pressure() const { return m_pressure; }
  * the given wavelength
  */
 double Material::cohScatterXSection(const double lambda) const {
-  (void)lambda;
-  return m_element.coh_scatt_xs;
+  UNUSED_ARG(lambda);
```

Comment:
They don't at the moment but they also pass the wavelength through, which is not yet used so if we started using that then we couldn't really cache anyway.